src/Microsoft.AspNet.OData.Shared/ODataQueryParameterBindingAttribute.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT License.  See License.txt in the project root for license information.
+
+using Microsoft.AspNet.OData.Query;
```

Comment:
what is using this?

Reply:
ODataQueryOptions.cs, ODataQueryOptionsOfTEntity.cs